patch/save_patches_old.txt

```diff
@@ -75,8 +74,7 @@ CreateKeyItemBitflags:
 	getfilebyte #1
 	call .loop_common_get
 	jumpz #1, .clear
-	increment #1
-	jumpz #1, .clear ; also clears 0xff
+	ifeq #1, 0xff, .clear
 	decrement #1
```

Comment:
This sequence isn't right now, since you left the `decrement` but not the `increment`.